lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -47,15 +47,19 @@ class LNUrlBloc with AsyncActionsHandler {
         injector.lightningLinks.linksNotifications,
         _lnurlInputController.stream,
 
-        // Process lightning-addresses in the clipboard only when the user manually 
+        // Process plain lightning-addresses in the clipboard only when the user manually
         // pastes them in and they show up in _lnurlInputController.stream.
-        injector.device.distinctClipboardStream
-            .where((l) => !isLightningAddress(l)),
+        // Also process lightning-address URIs.
+        injector.device.distinctClipboardStream.where((l) =>
+            !isLightningAddress(l) || l.toLowerCase().startsWith('lightning:')),
```

Comment:
should it be` !isLightningAddress(l) &&l.toLowerCase().startsWith('lightning:'))` ?
Or I am missing the condition here?
Previousely we filtered lightning addresses (emails) now it seems that we won't filter them if they prefixed with "lightning:"

Reply:
I've clarified the test in the latest commit.

       
        injector.device.distinctClipboardStream.where((l) {
          var result = true;
          if (isLightningAddress(l)) {
            result = false;
            result = l.toLowerCase().startsWith('lightning:');
          }
          return result;
        })